service/src/ocall_bridge/bridge_api.rs

```diff
@@ -195,6 +195,7 @@ pub trait WorkerOnChainBridge {
 #[cfg_attr(test, automock)]
 pub trait SidechainBridge {
 	fn propose_sidechain_blocks(&self, signed_blocks_encoded: Vec<u8>) -> OCallBridgeResult<()>;
+	fn store_sidechain_blocks(&self, signed_blocks_encoded: Vec<u8>) -> OCallBridgeResult<()>;
```

Comment:
I'm unsure if these two functions should be placed into the same trait, what's your take on this?

Reply:
Well just from the naming it seems to fit. What are your concerns why it shouldn't be here?

Reply:
Because apart from being a Sidechain and that they handle sidechainblocks, they have not much in common. They use a different struct each (proposer & storage) and are called from different ecall-threads. They are absolutely independent. Proposing can very well happen without any storage ever, and storing could happen without proposing as well. 

But if Sidechain related is enough to place them together, then I agree, that's the right place.

Reply:
Ok, I'll leave as it is now then. Thanks for your input.